firebase_admin/_auth_client.py

```diff
@@ -444,6 +444,7 @@ def generate_password_reset_link(self, email, action_code_settings=None):
 
         Raises:
             ValueError: If the provided arguments are invalid
+            EmailNotFoundError: If no user exists by the specified email address.
```

Comment:
Is there a specific reason to use "by" here?

I'd expect "for" probably, here and below.

Reply:
Thanks for catching. I copied and pasted blindly. It is fixed now here and elsewhere.